### PR TITLE
[fix](tracing) Fix wrong global variable init order of tracing log dir

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

File: gensrc/proto/internal_service.proto [hunk 1]
```diff
@@ -111,6 +111,15 @@ message PTabletWriterOpenResult {
     repeated PTabletLoadRowsetInfo tablet_load_rowset_num_infos = 2;
 };
 
+message PFinalizeCloseLoadChannelRequest {
+    required PUniqueId id = 1;
```
Comment (morrySnow) [inline gensrc/proto/internal_service.proto:115]: all member should be optional?

Comment (zclllyybb) [reply gensrc/proto/internal_service.proto]: sorry, some commits were brought up by mistake. fixed.

**Comment by zclllyybb** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by zclllyybb** [comment]
run buildall

**Review by pingchunzhang — APPROVED** [review]
LMGT

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**